packages/tools/src/eventListeners/touch/touchStartListener.ts

```diff
@@ -376,7 +386,7 @@ function _onTouchEnd(evt: TouchEvent): void {
 function _checkTouchTap(evt: TouchEvent): void {
   const currentTime = new Date().getTime();
   const startTime = state.startTime.getTime();
-  if (currentTime - startTime >= tapState.tapWindow) return;
+  if (currentTime - startTime > tapState.tapToleranceMs) return;
```

Comment:
I am still not convinced that both this line AND 401-402 are both needed. Why would they both be needed? They are practically identical considering line 435 resets the tap start time. I tried removing each and noticed no difference. Perhaps I am missing something?

Reply:
hmmmm yeah, I think you are right we can just rely on the touch start start time for tolerance. We can remove new Date() tracking for the tap state